src/segmantic/seg/monai_unet.py

```diff
@@ -292,6 +377,7 @@ def train(
     dataset: Union[Path, List[Path]] = [],
     image_dir: Path = None,
     labels_dir: Path = None,
+    cross_val: bool = False,
```

Comment:
adding this `cross_val` because of the different json formats is not so elegant. the number of function arguments is getting too long even without this one.